react-frontend/src/recognition-list-components/RecognitionCardComponent.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Row, Col } from "core-components/grid/GridComponent";
+import { Card } from "core-components/card/CardComponent";
+import { BsCircle } from "core-components/icon/icons";
+import ImageComponent from "core-components/image/ImageComponent";
+import RecognitionCardHeaderComponent from "recognition-list-components/RecognitionCardHeaderComponent";
+import RecognitionTextComponent from "recognition-list-components/RecognitionTextComponent";
+import HighFiveComponent from "shared-components/high-five-components/HighFiveComponent";
+
+const giveHighFive = () => {
+  //to to for give high five
+};
+
+const ImgD = styled.div`
+  border-radius: 20px;
+  height: auto;
+  width: auto;
+  overflow: hidden;
+`;
+
+const ImgM = styled.div`
+  border-radius: 20px;
+  height: auto;
+  width: auto;
+  overflow: hidden;
+`;
+
+const HighFive = styled.div`
+  position: absolute;
+  bottom: 50px;
+`;
+
+const RecognitionCardComponent = ({
+  given_by,
+  given_for,
+  given_at,
+  text,
+  core_value,
+  high_five_count,
+}) => (
+  <Card
+    className="my-4 mx-2 shadow p-3 mb-4 border border-secondary bg-light grey"
+    style={{ borderRadius: "36px" }}
+  >
+    <Card.Body>
+      <Row>
+        <Col sm="12" md="7">
+          <RecognitionCardHeaderComponent
+            given_for={given_for}
+            given_at={given_at}
+            core_value={core_value}
+          />
+          <ImgM className="bg-dark d-sm-block d-md-none mt-2">
+            <ImageComponent
+              src="https://i.picsum.photos/id/654/300/200.jpg"
+              alt="CoreValueImage"
+            />
+          </ImgM>
+          <RecognitionTextComponent text={text} given_by={given_by} />
+        </Col>
+        <Col className="d-none d-md-block">
```

Comment:
Read and understand bootstrap grid:

https://getbootstrap.com/docs/4.1/layout/grid/#how-it-works